js/commonjs/export_testdeps.js

```diff
@@ -13,12 +13,5 @@ goog.provide('jspb.ExportTestDeps');
 
 goog.require('goog.crypt.base64');
 goog.require('goog.testing.PropertyReplacer');
-goog.require('jspb.arith.Int64');
-goog.require('jspb.arith.UInt64');
-goog.require('jspb.BinaryEncoder');
-goog.require('jspb.BinaryDecoder');
-goog.require('jspb.BinaryWriter');
-goog.require('jspb.utils');
 
 exports.goog = goog;
-exports.jspb = jspb;
```

Comment:
`jspb` is available in the `google-protobuf` module. Exporting `jspb` here and subsequently using it in test will result in a lot of undefined symbol errors because the compiled `google-protobuf` is expecting obfuscated symbols.